apps/expo/src/app/_layout.tsx

```diff
@@ -1,13 +1,18 @@
+import { useSafeAreaInsets } from "react-native-safe-area-context";
 import { Stack } from "expo-router";
 import { StatusBar } from "expo-status-bar";
 
 import { TRPCProvider } from "~/utils/api";
 
 import "../styles.css";
 
+import { useColorScheme } from "nativewind";
+
 // This is the main layout of the app
 // It wraps your pages with the providers they need
 export default function RootLayout() {
+  const { colorScheme } = useColorScheme();
+  const insets = useSafeAreaInsets();
```

Comment:
Unused?

Reply:
colorScheme is used there, to pass the colours to expo-status-bar which uses RGB colours that we define in ``styles.css```.   I used RGB colours because we cannot read the css variables there https://github.com/marklawlor/nativewind/issues/682